routes.js

```diff
@@ -25,16 +26,16 @@ function fastifySwagger (fastify, opts, next) {
   })
 
   fastify.route({
-    url: '/documentation',
+    url: `${routePrefix}`,
     method: 'GET',
     schema: { hide: true },
-    handler: (request, reply) => reply.redirect('./documentation/')
+    handler: (request, reply) => reply.redirect(`.${routePrefix}/`)
```

Comment:
I'm not 100% sure how `reply.redirect()` works with the prefix. Open an issue on fastify if it does not work.